shared/actions/notifications.js

```diff
@@ -43,15 +43,15 @@ const setupEngineListeners = () => {
     })
   })
 
-  getEngine().setIncomingActionCreators('keybase.1.NotifyBadges.badgeState', ({badgeState}) =>
-    NotificationsGen.createReceivedBadgeState({badgeState})
-  )
-
-  getEngine().setIncomingActionCreators('stellar.1.notify.paymentNotification', refreshPayments)
-  getEngine().setIncomingActionCreators('stellar.1.notify.paymentStatusNotification', refreshPayments)
+  getEngine().setIncomingCallMap({
+    'keybase.1.NotifyBadges.badgeState': ({badgeState}) =>
+      Saga.put(NotificationsGen.createReceivedBadgeState({badgeState})),
+    'stellar.1.notify.paymentNotification': refreshPayments,
+    'stellar.1.notify.paymentStatusNotification': refreshPayments,
```

Comment:
@cjb note: this correctly fails flow as there is no accountID param in the paymentStatusNotification call